vertexai/main.ts

```diff
@@ -0,0 +1,29 @@
+import { initializeApp } from 'firebase/app';
```

Comment:
Should this have a copyright header?

Reply:
The other quickstarts don't, but better safe than sorry, added.